src/borg/testsuite/archiver/checks.py

```diff
@@ -17,7 +17,9 @@
 from .. import llfuse
 from .. import changedir, environment_variable
 from . import cmd, _extract_repository_id, open_repository, check_cache, create_test_files, create_src_archive
-from . import _set_repository_id, create_regular_file, assert_creates_file, RK_ENCRYPTION
+from . import _set_repository_id, create_regular_file, assert_creates_file, generate_archiver_tests, RK_ENCRYPTION
+
+pytest_generate_tests = lambda metafunc: generate_archiver_tests(metafunc, kinds="local,remote")  # NOQA
```

Comment:
Received a linting error: `E731 do not assign a lambda expression, use a def`


To avoid the linting error, I could do something like:

```
def pytest_generate_tests(metafunc):
    generate_archiver_tests(metafunc, kinds="local,remote,binary")

```

However, somewhat ironically, this would bring us almost full circle to what we were originally trying to avoid: re-defining `pytest_generate_tests` in each file 😂

Reply:
Well, both the assignment with lambda as well as the `def` define a function, so it is just 1 LOC difference. So guess just using the `def` is the simplest solution.

Reply:
or NOQA as you did it already it also fine.

Reply:
NOQA is fine to avoid this here.